tools/link.py

```diff
@@ -2113,12 +2113,17 @@ def create_worker_file(input_file, target_dir, output_file, options):
 
 def create_esm_wrapper(wrapper_file, support_target, wasm_target):
   wasm_exports = []
+  js_exports = []
   for f in settings.USER_EXPORTS:
     if f == '_main' and '__main_argc_argv' in settings.WASM_EXPORTS:
       wasm_exports.append('__main_argc_argv as main')
     else:
-      wasm_exports.append(shared.demangle_c_symbol_name(f))
+      if f in settings.WASM_EXPORTS:
+        wasm_exports.append(shared.demangle_c_symbol_name(f))
+      else:
+        js_exports.append(f)
```

Comment:
This could be "outdented"  above to an `elif` and `else`

Reply:
Nice